expression/column.go

```diff
@@ -344,6 +344,10 @@ func (col *Column) EvalInt(ctx sessionctx.Context, row chunk.Row) (int64, bool,
 		if val.IsNull() {
 			return 0, true, nil
 		}
+		if val.Kind() == types.KindMysqlBit {
+			val, err := val.GetBinaryLiteral().ToInt(ctx.GetSessionVars().StmtCtx)
+			return int64(val), err != nil, errors.Trace(err)
```

Comment:
Return `err` is enough because stack information has been wrapped.

Reply:
Ok